src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputData) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (String line : inputData.subList(1, inputData.size())) {
+            String[] partsOfData = line.split(",");
+            FruitTransaction.Operation fruitOperation =
+                    FruitTransaction.Operation.valueOf(partsOfData[0]);
+            String fruit = partsOfData[1];
+            int quantity = Integer.parseInt(partsOfData[2]);
+            fruitTransactions.add(new FruitTransaction(fruitOperation, fruit, quantity));
+        }
+        return fruitTransactions;
+    }
```

Comment:
It's good to see that you're skipping the first line of the input data, assuming it's a header. However, there should be a check to ensure that the inputData list is not empty before performing `inputData.subList(1, inputData.size())` to avoid potential `IndexOutOfBoundsException`.